### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by badersur**
In addition to these suggested fixes, I think the first sentence in "Latest Features" section could be removed...

### CODE REVIEW

File: docs/csharp/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The C# guide provides a wealth of information about the C# language. This site h
...
 
 * For developers new to C#: 
-    - If you've done development before, but are new to C#, read the [tour of the C# language](tour-of-csharp/index.md). You will lern the basic syntax and structure for the language, and you can use the language tour to contrast C# with other languages you've used. You can also browse the [tutorials](tutorials/index.md) to try basic C# programs. Or, check out the [C# Concepts](concepts.md) to learn the features of the C# language in depth.
+    - If you've done development before, but are new to C#, read the [tour of the C# language](tour-of-csharp/index.md). You will learn the basic syntax and structure for the language, and you can use the language tour to contrast C# with other languages you've used. You can also browse the [tutorials](tutorials/index.md) to try basic C# programs. Or, check out the [C# Concepts](concepts.md) to learn the features of the C# language in depth.
```
Comment (badersur) [inline docs/csharp/index.md]: Maybe "learn about"?!

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/index?branch=pr-en-us-1055](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/index?branch=pr-en-us-1055)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C21%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609210959513094-1055%5Cworkflow_report.html).

File: docs/csharp/index.md (hunk 1 above)
Comment (mairaw) [inline docs/csharp/index.md]: I think it's ok as-is

**Review by mairaw — APPROVED** [review]
Thank you for fixing these issues. LGTM @badersur!

**Pull request merged**